estate/models/estate_property.py

```diff
@@ -0,0 +1,38 @@
+from odoo import fields, models
+from dateutil.relativedelta import relativedelta
+
+
+class EstateProperty(models.Model):
+    _name="estate.property"
+    _description="Estate Model"
+
+    name = fields.Char(required=True)
+    description = fields.Text()
+    postcode=fields.Char()
+    date_availability = fields.Date(
+        copy=False,
+        default= (fields.Date.today() + relativedelta(months=3))
+        # default=fields.Date.add(fields.Date.today() , months=3)
+    )    
+    expected_price = fields.Float(required=True)
+    selling_price = fields.Float(readonly=True,copy=False)
+    bedrooms = fields.Integer(defualt=2)
+    living_area = fields.Integer()
+    facades = fields.Integer()
+    is_garage = fields.Boolean()
+    is_garden = fields.Boolean()
+    garden_area = fields.Integer()
+
+    garden_orientation = fields.Selection(
+        selection=[('north', 'North'), ('south', 'South'), ('east', 'East'), ('west', 'West')],
+    )
+    active = fields.Boolean(default = 'True')
+    status = fields.Selection(
+        selection=[('new', 'New'), ('offer received', 'Offer Received'), ('offer accepted','Offer Accepted'), ('sold','Sold'), ('cancelled','cancelled')],
+        default = 'new'
+    )
+    property_type_id= fields.Many2one('estate.property.type',string="Property Type")
+    buyer= fields.Many2one('res.partner',string="Buyer",copy=False)
+    seller= fields.Many2one('res.users',string="Seller",default=lambda self: self.env.user)
+    tag_ids = fields.Many2many('estate.property.tag',string="Tags")
+    offer_ids = fields.One2many('estate.property.offer',"property_id",string="Offer")
```

Comment:
```suggestion
    tag_ids = fields.Many2many('estate.property.tag', string="Tags")
    offer_ids = fields.One2many('estate.property.offer', "property_id", string="Offer")
```
add a space between to make it clear in any field